cloud_build/verify_provenance.sh

```diff
@@ -29,9 +34,30 @@ popd
 # Note: jq is used in order to obtain the full artifact registry url from
 # the provenance metadata.
 echo "Verifying the provenance is valid and correct..."
-FULLY_QUALIFIED_DIGEST=$(cat $1 | \
-  jq -r .image_summary.fully_qualified_digest)
+echo "Checking for source-uri of $SOURCE_URI"
```

Comment:
Can we add a TODO to remove this when the change is stably rolled out?

Reply:
Added!